### ISSUE TITLE
Comments in match statement throw error

### ISSUE THREAD

**Issue by mklein994** [prettier/plugin-php#1956]
**@prettier/plugin-php v0.18.3**
[Playground link](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14AsLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8M+VBZhqYAGpIlWfCDoA6AIwgWMCG3iyVAYXQBbH2wAFAAkA-BdBQxAOVwBjKABlGABPNWswbAgAWgAmJxBoAEdBeGgA12V-PTkFEGIRCBgytgq2KoMAXxYoCngYdBQ0eCxJUDZsGDaRTtrMMEEJvBAAHgB+YzAAHUwtqUFMGIxMAAIpeAAvM45BC4AKYhgjgBJ4AEot4C2jr6OiEiP-GAxahHG4MF5HD7Hb7QgD0ACojjR4NgjsijmwjjFfD4oH84TDPtDvs8jgBSI72ACsRwAvAA+I4bECnC5XC5MpiEolPeBko4AZlpDKZLLOHK5RJJ5Op9MZUWuYqcEuhFCgMkEsgesuenKhX3aR1sR2CAQA+gBRZAAGQA3Ft2iB2kA)

**Input:**
```php
<?php

function fizzbuzz(int $i)
{
    print match (0) {
        /* this is a comment */
        $i % 15 => "fizzbuzz",
        $i % 3 => "fizz",
        $i % 5 => "buzz",
        default => $i,
    } . PHP_EOL;
}
```

**Output:**
```php
Comment "/* this is a comment */" was not printed. Please report this error!
```

Of note, it doesn't matter what kind of comment, or where the comment is (end of line, before/after a pattern, etc.). Every scenario throws an exception.

**Comment by jaulz** [comment]
Yep, can confirm the same from my side.

**Comment by juandelperal** [comment]
Yep, same here

**Comment by villermen** [comment]
While not ideal, I addressed this in the meantime by moving the comment to either before the entire statement or after one of the arrows:

```php
<?php

function fizzbuzz(int $i)
{
    /* this is a comment */
    print match (0) {
        $i % 15 => "fizzbuzz",
        $i % 3 => "fizz",
        $i % 5 => /* this is another comment */ "buzz",
        default => $i,
    } . PHP_EOL;
}
```

**Comment by czosel** [comment]
Fixed in `v0.19.7` :tada:
